scripts/k8s_e2e_node_windows.ps1

```diff
@@ -0,0 +1,74 @@
+param (
+    [string]$repoName = "kubernetes",
+    [string]$repoOrg = "kubernetes",
+    [string]$pullRequestNo,
+    [string]$pullBaseRef = "master"
+)
+
+$LogsDirPath = "c:/Logs"
+$RepoPath = "c:/$repoName"
+$RepoURL = "https://github.com/$repoOrg/$repoName"
+$LocalPullBranch = "testBranch"
+
+function Prepare-LogsDir {
+    
+    mkdir $LogsDirPath
+
+}
+
+function Clone-TestRepo {
+    Write-Host "Cloning $repoName repo"
+    git clone --branch $pullBaseRef $RepoURL $RepoPath --depth=1
+    if (($pullRequestNo -ne $null) -and ($pullRequestNo -ne "")) {
+        Write-Host "Pulling PR $pullRequestNo changes"
+        cd $RepoPath
+        git fetch origin refs/pull/$pullRequestNo/head:$LocalPullBranch
+        git checkout $LocalPullBranch
+        if (! $?) {
+            Write-Host "Failed to pull PR $pullRequestNo changes. Exiting"
+            exit
+        }
+    }
+}
+
+function Build-Kubelet {
+    Write-Host "Building kubelet"
+
+    Push-Location "$RepoPath"
+    go build -o .\_output\kubelet.exe .\cmd\kubelet\
```

Comment:
Can't you do `make WHAT=cmd/kubelet` or is `make` not installed?

Reply:
I don't think we have `make` installed here and also the make targets might also assume bash is installed.